localization/english/dvg_flavor_atlantia_l_english.yml

```diff
@@ -0,0 +1,67 @@
+﻿l_english:
+
+ #Decisions
+
+ #Events
+ dvg_atlantia.1.t: "The Future of Atlantia"
+ dvg_atlantia.1.d: "The time has come to decide who should govern Atlantia."

```

Comment:
Use [ROOT.GetCountry.GetName] when using the root country name in event. This makes it light up and hoverable. This brings our events at the same level of vanilla ones.

You can also do the same with cultures. If you get the culture in 'immediate' and save it as a scope.
[SCOPE.sPop('scope_name').GetCulture.GetName]

This is only for description and flavor, not for options or titles.